doc-content/enterprise-only/bpmn/bpmn-user-task-permission-ref.adoc

```diff
@@ -0,0 +1,93 @@
+[id='bpmn-user-task-permission-matrix-ref']
+= BPMN2 task permission matrix in process designer
+
+The user task permission matrix summarizes the actions that are allowed for specific user roles. The user roles are as follows:
+
+* Potential owner: User who can claim the task, which was claimed earlier and has been released and forwarded. The tasks with `Ready` status can be claimed, and the potential owner becomes the actual owner of the task.
```

Comment:
Instead of **has been released**  you can try this: 
Suggestion: ...which was claimed earlier and **it is released** and forwarded.

Reply:
Done, thanks